modules/statistics/php/NDB_Form_statistics.class.inc

```diff
@@ -785,55 +795,228 @@ class NDB_Form_Statistics extends NDB_Form
+                            SUM($unspec_scans) AS total_unsp
+                	    FROM
+                	        mri_parameter_form m
+                	        JOIN flag f USING (CommentID)
+                	        JOIN session s ON (f.SessionID=s.ID)
+                	        JOIN candidate c ON (c.CandID=s.CandID)
+                	    WHERE 
+                	        s.Current_stage <> 'Recycling Bin'
+                	        AND f.Administration<>'NONE'
+                	        AND s.Active='Y'
+                	        AND c.Active='Y'
+                	        AND f.CommentID NOT LIKE 'DDE&'
+                	        AND s.CenterID <> '1'
+							AND (c.ProjectID IS NULL OR c.ProjectID=$pid)
+                	        $suproject_query
+                	    GROUP BY
+                	        s.SubprojectID,
+                	        c.CenterID,
+                	        s.Visit_label)";
+            } else {
+                $Pre_Result_One
+                    = "CREATE TEMPORARY TABLE IF NOT EXISTS temp_totals AS (
+            	        SELECT 
+            	            s.SubprojectID,
+            	            c.CenterID,
+            	            s.Visit_label AS VLabel,
+            	            SUM($no_scans) AS total_no,
+            	            SUM($partial_scans) AS total_part,
+            	            SUM($complete_scans) AS total_comp,
+                            SUM($unspec_scans) AS total_unsp
+            	        FROM
+            	            mri_parameter_form m
+            	            JOIN flag f USING (CommentID)
+            	            JOIN session s ON (f.SessionID=s.ID)
+            	            JOIN candidate c ON (c.CandID=s.CandID)
+            	        WHERE 
+            	            s.Current_stage <> 'Recycling Bin'
+            	            AND f.Administration<>'NONE'
+            	            AND s.Active='Y'
+            	            AND c.Active='Y'
+            	            AND f.CommentID NOT LIKE 'DDE&'
+            	            AND s.CenterID <> '1'
+            	            $suproject_query
+            	        GROUP BY
+            	            s.SubprojectID,
+            	            c.CenterID,
+            	            s.Visit_label)";
+            }
+            $PRO_params['no_scans']        = $no_scans;
+            $PRO_params['partial_scans']   = $partial_scans;
+            $PRO_params['complete_scans']  = $complete_scans;
+            $PRO_params['suproject_query'] = $suproject_query;
+            $PRO_params['Param_Project']   = $Param_Project;
+            $PRO_params['unspec_scans']    = $unspec_scans;
+
+            $Pre_Result_Two   = "CREATE TEMPORARY TABLE IF NOT EXISTS temp_comp AS (
+                    SELECT
+                        t.SubprojectID,
+                        t.CenterID,
+                        t.VLabel,
+                        \"Complete\" AS Subcat,
+                        t.total_comp AS val
+                    FROM temp_totals t)";
```

Comment:
You can use the literal quote type (`'`) instead of the interpolated type (`"`) in order to avoid having to escape the quotation marks inside the string.